shell/platform/fuchsia/flutter/flatland_platform_view.cc

```diff
@@ -75,10 +75,14 @@ void FlatlandPlatformView::OnGetLayout(
   view_logical_size_ = {static_cast<float>(info.logical_size().width),
                         static_cast<float>(info.logical_size().height)};
 
-  // TODO(fxbug.dev/94000): Set device pixel ratio.
+  float pixel_ratio = 1.0f;
+  if (info.has_device_pixel_ratio()) {
+    view_pixel_ratio_ = info.device_pixel_ratio().x;
+    pixel_ratio = *view_pixel_ratio_;
+  }
 
   SetViewportMetrics({
-      1,                              // device_pixel_ratio
+      pixel_ratio,                    // device_pixel_ratio
```

Comment:
We can exercise this via FlatlandPlatformViewTests.SetViewportMetrics

Reply:
Done